index.html

```diff
@@ -0,0 +1,138 @@
+    <meta name="author" content="Arthur Guzman" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/public/favicon-32x32.svg" />
+    <script
+      src="https://kit.fontawesome.com/27e53b408d.js"
+      crossorigin="anonymous">
+    </script>
+    <link rel="stylesheet" href="css/home/home.css" />
+    <title>Web Design for Everybody: Basics of Web Development and Coding | CodeClub</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="index.html" class="link"><img src="public/logo.svg" alt="code club icon"></a>
+            <ul>
+                <li>
+                    <a href="about.html" class="link">About</a>
+                </li>
+                <li>
+                    <a href="#" class="link">Program</a>
+                </li>
+                <li>
+                    <a href="#" class="link">Join for Free</a>
+                </li>
+            </ul>
+            <i class="fas fa-bars hamb"></i>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <p class="headline__greet">"Hello World!"</p>
+            <h1 class="headline__title">Web Design for Everybody: Basics of Web Development and Coding</h1>
+            <div class="headline__dsc">
+                <p>Learn to Design and Create Websites. Build a responsive and accessible web portfolio using HTML5, CSS3, and JavaScript.</p>
+            </div>
+            <p class="headline__date">2021.07.21(TUE) ~ 22(THU)</p>
+        </section>
+        <section class="program">
+            <h2 class="program__title">Main Program</h2>
+            <hr>
+            <ul>
+                <li>
+                    <img src="public/Icon-1.svg" alt="lecture icon">
+                    <h3>Lecture</h3>
+                    <p class="program__p">Start the course and speech about web design for future.</p>
+                </li>
+                <li> 
+                    <img src="public/Icon-2.svg" alt="lecture icon">
+                    <h3>First Stage</h3>
+                    <p class="program__p">Start HTML-CSS course emphasizing semantic rules.</p>
+                </li>
+                <li>
+                    <img src="public/Icon-3.svg" alt="lecture icon">
+                    <h3>Forum</h3>
+                    <p class="program__p">Time to share your doubts about what you are learning.</p>
+                </li>
+                <li>
+                    <img src="public/Icon-2.svg" alt="lecture icon">
+                    <h3>Second Stage</h3>
+                    <p class="program__p">Start the course of JavaScript and Responsive Design</p>
+                </li>
+                <li>
+                    <img src="public/Icon-1.svg" alt="lecture icon">
+                    <h3>Closing Party</h3>
+                    <p class="program__p">Closing the course and exchange of opinions about it.</p>
+                </li>
+            </ul>
+        </section>
+        <section class="presenter">
```

Comment:
The section should be designed like the below screenshot and take advantage of css flex
![right4](https://user-images.githubusercontent.com/58376020/123713741-5c862300-d875-11eb-962f-dd56b336bcfd.png)